### ISSUE TITLE
Auto populate holiday / sickness rows of timesheets from absence records

### ISSUE THREAD

**Issue by aharwood2** [UoMResearchIT/CapX#627]
Might be nice to auto-populate these rows but I think we need to change the absence models first to allow half days #435 and also distinguish between absence types.

**Comment by ajerrison-manchester** [comment]
I seem to remember that there was some concern about privacy when I mentioned that sickness was being recorded on CapX at the RIT team meeting (when I was presenting on your behalf a few months ago). That was somewhat alleviated by the fact that there was no category of absence, it was just a way of saying that someone wasn't available. So it probably warrants  consultation within the RSE team at least.

**Comment by aharwood2** [comment]
Absolutely. I think that was largely because we didn't have line manager relationships. Now we do, it is possible that we can protect the nature of the absence from the general manager group. There also wouldn't be anything needed besides what we need to determine the timesheet category to book the absence to.

**Comment by ianhinder** [comment]
Do we need some sort of extra level of vigilance (IGRR?) if we are storing things which might be considered HR-related? Absence itself sounds like something that a very risk-averse lawyer-type may object to, and nature of absence something that might raise eyebrows more widely, even if it is intended to be protected by authorisation for line managers only.

**Comment by PhilBradbury** [comment]
Someone being absent on a date is not a problem. Revealing why that person is off is though. It is fine to show that someone is not available at work -- indeed it could be fairly critical. At least this was the University's stance on this *cough* years ago when I wrote a system for The Faculty of Arts IT Support Unit. 

We'd need to store half days and whether it was sickness or annual leave (well, for as many types are in the system to be booked to, I guess) and this could be protected to be viewed by Managers only, but auto populating it would be tricky due to the differing number of hours that would need to be entered which would depend on the person's work pattern (FT / PT / compressed hours etc.).
